ecommerce/frontend/src/components/ChooseListModal.js

```diff
@@ -0,0 +1,54 @@
+import './ChooseListModal.less'
+
+import React, { useEffect, useState } from 'react'
+import { List } from 'antd'
+import { sleep } from '../utils'
+
+export const ChooseListModal = ({ onChooseList }) => {
+    const [lists, setLists] = useState([])
+    const [loading, setLoading] = useState(true)
+
+    useEffect(() => {
+        async function fetch() {
+            try {
+                setLoading(true)
+                await sleep(1000)
+
+                setLists([
+                    {
+                        id: 1,
+                        name: 'Electronics',
+                    },
+
+                    {
+                        id: 2,
+                        name: 'Thanksgiving',
+                    },
+                    {
+                        id: 3,
+                        name: 'Blackfriday',
+                    },
+                ])
+            } catch (error) {
+            } finally {
+                setLoading(false)
+            }
+        }
+
+        fetch()
+    }, [])
+
+    const onListClick = list => () => onChooseList(list)
+
+    return (
+        <List loading={loading} size="large">
+            {lists.map((list, ix) => {
+                return (
+                    <List.Item className={'choose-list-modal-list'} onClick={onListClick(list)} key={list.id ?? ix}>
+                        {list.name ?? `Example list name ${ix}`}
```

Comment:
You can add heart button next to each list item in the popup in case customer wants to add that product to multiple wishlists.
`<Button onClick={onAddToListClicked(ix)} size="large" icon={<HeartOutlined />} />`

Reply:
Okay it seems that both you and @ozdenizdolu want the user be able to add to multiple lists at once.. guess I'll do it.